src/lib.rs

```diff
@@ -53,7 +53,7 @@ pub fn get_zones_and_stats(
     info!("reading pbf done.");
 
     let mut zones = vec![];
-    let mut stats = CosmogonyStats::default();
```

Comment:
this stat obj is still used ?

Reply:
yes, it is returned by the current function (`get_zones_and_stats`)